rootless.md

```diff
@@ -8,6 +8,8 @@ Contributors are more than welcomed to help with this work.  If you decide to ca
   * The kernel does not allow processes without CAP_NET_BIND_SERVICE to bind to low ports.
   * You can modify the `net.ipv4.ip_unprivileged_port_start` sysctl to change the lowest port.  For example `sysctl net.ipv4.ip_unprivileged_port_start=443` allows rootless Podman containers to bind to ports >= 443.
   * A proxy server, kernel firewall rule, or redirection tool such as [redir](https://github.com/troglobit/redir) may be used to redirect traffic from a privileged port to an unprivileged one (where a podman pod is bound) in a server scenario - where a user has access to the root account (or setuid on the binary would be an acceptable risk), but wants to run the containers as an unprivileged user for enhanced security and for a limited number of pre-known ports.
+* As of podman 5.0 pasta is the default networking tool. Since pasta copies the ip address of the main interface, connections to that ip from containers does not work. This means that unless you have more than one interface, inter-container connections cannot be made without explicitly passing pasta network configuration, either in ~.config/containers/containers.conf or at runtime.
```

Comment:
```suggestion
* As of podman 5.0 pasta is the default networking tool. Since pasta copies the ip address of the main interface, connections to that ip from containers does not work. This means that unless you have more than one interface, inter-container connections cannot be made without explicitly passing pasta network configuration, either in `containers.conf` or at runtime.
```